install.sh

```diff
@@ -32,26 +32,21 @@ fi
 # 2.scp resources
 echo "2.scp resources"
 sh ${workDir}/script/scp-hosts.sh
-if [ $? -eq 0 ]
-then
+if [ $? -eq 0 ];then
 	echo 'scp copy completed'
 else
 	echo 'scp copy failed to exit'
 	exit 1
 fi
 
-
 # 3.stop server
 echo "3.stop server"
 sh ${workDir}/script/stop-all.sh
 
-
 # 4.delete zk node
 echo "4.delete zk node"
-
 sh ${workDir}/script/remove-zk-node.sh $zkRoot
 
-
 # 5.startup
 echo "5.startup"
-sh ${workDir}/script/start-all.sh
+sh ${workDir}/script/start-all.sh
```

Comment:
Please keep file end with new line 
```suggestion
sh ${workDir}/script/start-all.sh

```

Reply:
@tracehh could you have time to add new line in the end of this file?